mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -256,6 +256,48 @@ def Rock_GeneralGemmParamsAttr : Rock_Attr<"GeneralGemmParams", [RockTuningParam
   }];
 }
 
+def Rock_AttnPerfConfig : Rock_Attr<"AttnPerfConfig", [RockTuningParamAttrInterface]> {
+  let mnemonic = "attn_perf_config";
+  let description = [{
+    The perf configs for rock.attention operator.
+  }];
+  let parameters = (ins
+    "int64_t":$version,
```

Comment:
`version` is a property of the printed output and parsed input.

If it's in our repo, it should be upgraded to the new/only form.

There shouldn't be any "old version" handling past the parser

Reply:
Ack. will change this.

Reply:
done